src/pages/LandingPanel.vue

```diff
@@ -10,10 +10,8 @@ export default defineComponent({});
   </main>
 </template>
 
-<style lang="scss">
-.landing-panel {
-  display: flex;
-  flex-direction: column;
-  gap: 6.4rem;
-}
+<style lang="scss" scoped>
+  .landing-panel {
+    min-height: calc(100vh + 1px);
```

Comment:
Why was this needed?

Reply:
What I mean is why `+ 1px`

Reply:
That's a leftover code and I'm sorry for leaving it here.
As far as I remember the purpose was to have a quick fix of screen-jumping behaviour while data to be displayed on the screen was re-fetching. Loading element shown before displaying the fetched content was not big enough to make screen scrollable. I didn't want to make every screen on `overscroll: visible` or something like that so that was the idea for a moment.
Anyway, I figured out how to deal with that jumpy behaviour programmatically with correct `v-if` statement in the template and the snipped you asked about is no longer needed. Will be deleted.

Reply:
Awesome 👍